doc/project/code_flow.rst

```diff
@@ -6,38 +6,58 @@ Code Flow and Branches
 Introduction
 ************
 
-The zephyr Git repository has three types of branches:
+The Zephyr Git repository has three types of branches:
```

Comment:
No, this is done on purpose. The convention that David Kinder established is that "the zephyr repository" means https://github.com/zephyrproject-rtos/zephyr. If we use uppercase ("Zephyr"), we are referring to the project more widely. Please remove this change and do not introduce more inconsistencies. CC @kartben please take note.